.github/workflows/ci-fast.yml

```diff
@@ -86,3 +86,16 @@ jobs:
       python-version: ${{ matrix.python-version }}
       test_environment: ${{ matrix.test_environment }}
     secrets: inherit
+  test-generate-test-duration:
+    strategy:
+      matrix:
+        os: [ubuntu-dind-runners]
+        python-version: ['3.10']
+        test_environment: [default]
```

Comment:
does this need a matrix? can't we just use the OS + Python version in the `with` params below directly?